### PR TITLE
update driver handle

### PR DESCRIPTION
**Pull request by adrianmaciuc**

### CODE REVIEW

File: .idea/misc.xml
Comment (AdrianCiuciui) [inline .idea/misc.xml:1]: I'm not using coretto, I prefer java sdk
Please remove this misc.xml file, or add it to gitignore

File: .idea/jarRepositories.xml
Comment (AdrianCiuciui) [inline .idea/jarRepositories.xml:1]: pls add this file to gitignore also

File: .idea/vcs.xml
Comment (AdrianCiuciui) [inline .idea/vcs.xml:1]: pls add this to gitignore also

File: .idea/.gitignore
Comment (AdrianCiuciui) [inline .idea/.gitignore:1]: pls add to gitignore

File: .idea/misc.xml
Comment (adrianmaciuc) [reply .idea/misc.xml:1]: files removed

File: .idea/jarRepositories.xml
Comment (adrianmaciuc) [reply .idea/jarRepositories.xml:1]: files removed

File: .idea/.gitignore
Comment (adrianmaciuc) [reply .idea/.gitignore:1]: files removed

**Review by AdrianCiuciui — APPROVED** [review]

**Comment by AdrianCiuciui** [comment]
Thank you for the improvement on the handling of the driver. It's much cleaner now.

**Pull request merged**